### ISSUE TITLE
Use bulk permission-policy lookups to optimise listing views

### ISSUE THREAD

**Issue by gasman** [wagtail/wagtail#10443]
Follow-up to #10442 - once the `annotate_with_permissions` method on PermissionPolicy is implemented, ensure that it's used on all relevant views. Ideally this should be implemented through the generic listing view `wagtail.admin.views.generic.models.IndexView`, with other listings ported to use that if they aren't already doing so.

**Comment by infraredCoding** [comment]
Hey @gasman . Is this issue is being worked on? If not I'd love to pick it up.

**Comment by gasman** [comment]
Hi @infraredCoding - thanks for your interest! This was part of a wider piece of work for Wagtail 5.1 around improving performance of permission checks (see https://github.com/wagtail/wagtail/pull/10470, https://github.com/wagtail/wagtail/pull/10521, https://github.com/wagtail/wagtail/pull/10562), and during that work we found that this wasn't actually necessary - rather than performing more complex queries to perform multiple checks at once, we got better results by caching previous results on the request object. So, I'll close this as no longer needed.
